### DISCUSSION TITLE
Branch prefix and suffix

### DISCUSSION THREAD

**Discussion by Ashijo** [gitbutlerapp/gitbutler#2604] [Ideas]
At my job we have requirement for big prefix in our branch name. like ```users/[my-name]/features/[us-id]_[custom-name]```
Every time I create a branch I have to re-write ```users/[my-name]/features/``` . 
We also have other conventions for branch release and hotfixes...

I have aliases that do it for me, but, it would be cute to have a branch prefix functionality available one day :)

**Comment by schacon** [comment]
Here is a tracking issue for this request: https://github.com/gitbutlerapp/gitbutler/issues/2717

We've been talking internally about the best way to approach this. You can track that issue if you want updates on it's status. Thanks!
